app/models/testimonial.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: testimonials
+#
+#  id           :bigint           not null, primary key
+#  content      :string
+#  discarded_at :datetime
+#  name         :string           not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+# Indexes
+#
+#  index_testimonials_on_discarded_at  (discarded_at)
+#
+class Testimonial < ApplicationRecord
+  include Discard::Model
+  has_one_attached :testimonial_image
```

Comment:
In this case it is not necessary to use `testimonial_image`:
```suggestion
  has_one_attached :image
```